src/main/java/design/MazeDesignerInteractor.java

```diff
@@ -0,0 +1,47 @@
+package design;
+
+import entities.DesignableMaze;
+
+public class MazeDesignerInteractor implements MazeDesignerInputBoundary {
+
+    DesignableMaze dm;
+
+    public void newMaze(){
+        dm = new DesignableMaze(11, 17);
```

Comment:
when you add `private RandomizedPrim mg;` as an instance attribute don't forget to reset mg with:

`this.mg = new RandomizedPrim(dm)`